README.md

```diff
@@ -1,57 +1,189 @@
--------
-### ⚠️  PLEASE READ THE [INSTRUCTIONS](/INSTRUCTIONS.md) FOR GUIDELINES ON HOW TO START YOUR PACKAGE.
-> Don't forget to remove this warning while updating this README.
--------
-
-# {package-name}
+# react-accordion
 
 [![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
 
-[npm-url]:https://npmjs.org/package/@moxy/{package-name}
-[downloads-image]:https://img.shields.io/npm/dm/@moxy/{package-name}.svg
-[npm-image]:https://img.shields.io/npm/v/@moxy/{package-name}.svg
-[build-status-url]:https://github.com/moxystudio/{package-name}/actions
-[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/{package-name}/Node%20CI/master
-[codecov-url]:https://codecov.io/gh/moxystudio/{package-name}
-[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/{package-name}/master.svg
-[david-dm-url]:https://david-dm.org/moxystudio/{package-name}
-[david-dm-image]:https://img.shields.io/david/moxystudio/{package-name}.svg
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/react-accordion
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-accordion.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-accordion.svg
+[build-status-url]:https://github.com/moxystudio/react-accordion/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-accordion/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-accordion
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-accordion/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-accordion
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-accordion.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-accordion?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-accordion.svg
 
-{package-drescription}
+A react component that renders an accessible accordion.
 
 ## Installation
 
 ```sh
-$ npm install @moxy/{package-name}
+$ npm install @moxy/react-accordion
 ```
 
 This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
 
 ## Motivation
 
-{package-motivation}
+An accordion allows users to toggle the display of sections of content. This package was created to make it easier to implement an accordion with accessible navigation.
```

Comment:
Small nitpick, but the end of the first sentence would sound better with `content sections` instead of `sections of content`.

Reply:
Sure, I will change it 👍